sunpy/net/tests/strategies.py

```diff
@@ -13,8 +13,10 @@
 
 @st.composite
 def Times(draw, max_value, min_value):
-    time = datetimes(max_value=max_value, min_value=min_value)
-    time = Time(draw(time), format='datetime')
+    time = one_of(datetimes(max_value=max_value, min_value=min_value),
+                  sampled_from((Time('2008-12-31T23:59:60'),
```

Comment:
I think it might be worth moving this out into a separate strategy and then pulling it back in here. I am thinking this because I can foresee needing to write specific tests that check the behaviour of leap seconds, so having a separate strategy for just leap seconds means we can write tests specific to them as well as general ones.